routers/api/v1/repo/topic.go

```diff
@@ -164,7 +169,7 @@ func AddTopic(ctx *context.APIContext) {
 	}
 
 	// Prevent adding more topics than allowed to repo
-	topics, err := models.FindTopics(&models.FindTopicOptions{
+	topics, _, err := models.FindTopics(&models.FindTopicOptions{
```

Comment:
This should be CountTopics

Reply:
do you mean separate to two functions?

Reply:
Look at line 180 below.

Reply:
it doesnt mather if we use len or the returned value in this case